resources/folder.rb

```diff
@@ -1,6 +1,7 @@
 # To learn more about Custom Resources, see https://docs.chef.io/custom_resources.html
 property :host,           String,   default: 'localhost'
 property :port,           Integer,  default: 3000
+property :url_path_prefix, [String, nil], default: nil
```

Comment:
``Properties have a nil value by default so there is no need to set the default value to nil.`` This needs to be addressed as well.